Traceback (most recent call last):
  File "/HOME/nscc-gz_jiangli/virtualenv/ht_celery/simtask.py", line 18, in <module>
    raise Exception( "test exception " )
Exception: test exception 
